etc/conf/macos-patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cylc/flow/hostuserutil.py b/cylc/flow/hostuserutil.py
+index 21e51735e..17917b8fc 100644
+--- a/cylc/flow/hostuserutil.py
++++ b/cylc/flow/hostuserutil.py
+@@ -113,7 +113,7 @@ class HostUtil:
+         """Return the extended info of the current host."""
+         if target not in self._host_exs:
+             if target is None:
+-                target = socket.getfqdn()
++                target = socket.gethostname()
```

Comment:
```suggestion
                 target = socket.getfqdn()
                 target = socket.gethostname()
```

Reply:
It's a patch file so these +/- symbols tell the `patch` command which lines to add/remove.

Reply:
doh, my brain!